tick/plot/plot_stem.py

```diff
@@ -1,12 +1,13 @@
 import numpy as np
+import warnings
 
 
 def stem(y: np.ndarray, show=True, title=None, x_range=None, y_range=None):
     """Stem plot using bokeh
 
     Parameters
     ----------
-    y : `np.ndarray`, shape=(n_coeffs,)
+    y : `numpy.ndarray`, shape=(n_coeffs,)
```

Comment:
If think that using np.ndarray does not link to the numpy doc in sphinx's html while numpy.ndarray does, but it needs to be confirmed :)